nose_launchable/plugin.py

```diff
@@ -52,30 +49,42 @@ def configure(self, options, conf):
         super(Launchable, self).configure(options, conf)
 
         self.subset_enabled = options.subset_enabled or False
+        self.record_only_enabled = options.record_only_enabled or False
+
         self.build_number = options.build_number or os.getenv(BUILD_NUMBER_KEY)
         self.subset_target = options.subset_target
-
         self.subset_options = options.subset_options
 
-        self.record_only_enabled = options.record_only_enabled or False
-
         if not (self.subset_enabled or self.record_only_enabled):
+            # we didn't get activated
+            return
+
+        if os.environ[LaunchableClientFactory.TOKEN_KEY] is None:
+            logger.warning("%s not set. Deactivating"%LaunchableClientFactory.TOKEN_KEY)
             return
```

Comment:
I thought about defining a custom exception type that can be used in nose-launchable to report a graceful warning and bail out, but opted for something quicker even at the cost of duplicating the check.